emulator/src/extensions/emu_ext.h

```diff
@@ -94,6 +96,99 @@ typedef enum
     EMU_SCROLL_RIGHT = 0x40,
 } mouseButton_t;
 
+typedef enum
+{
+    EMU_MOD_NONE  = 0x00,
+    EMU_MOD_ALT   = 0x01,
+    EMU_MOD_CTRL  = 0x02,
+    EMU_MOD_SHIFT = 0x04,
+    EMU_MOD_SUPER = 0x08,
+} modKey_t;
+
+/**
+ * @brief Normalized key-codes across all platforms for use by emulator extensions
+ * These are just the CNFG keycodes renamed so extensions don't have to include rawdraw directly.
```

Comment:
Eeh, how bad is including rawdraw directly? I look at this and see one more thing to maintain (even if keyboards aren't likely to change any time soon).

Reply:
@dylwhich I'm good to merge if you stand by the normalized key codes